### PR TITLE
Remove APP_RELEASE environment variable

### LINKED ISSUE #796: Docs incorrectly state that RESIN_APP_RELEASE is available inside a container
https://docs.resin.io/learn/develop/runtime/

The docs state that the `RESIN_APP_RELEASE` env var is available inside the running container. I believe that for post multicontainer OS versions this is not the case.

### PR DESCRIPTION
**Pull request by adipascu**
This fixes #796

### CODE REVIEW

**Comment by adipascu** [comment]
I have no idea why CI failed, I prefer to not log into ci.balena-dev.com.
Can somebody resubmit/merge this?

File: pages/learn/develop/runtime.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For many applications, the code running in your container will need some way to
 
 ### Environment variables
 
-Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:
+Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:P
```
Comment (chrisys) [inline pages/learn/develop/runtime.md]:
```suggestion
Inside your running container, you'll have access to a number of `{{ $names.company.allCaps }}_` namespaced environment variables, which provide information from the system outside the container:
```

Comment (adipascu) [reply pages/learn/develop/runtime.md]: @chrisys Its done, no idea how that one got in. Thanks for maintaining balena!

**Comment by adipascu** [comment]
@chrisys done

**Comment by nazrhom** [comment]
@resin-ci retest

**Pull request merged**